src/modelValidator.js

```diff
@@ -373,6 +377,25 @@ class ModelValidator {
       }
     }
   }
+
+  _checkFactUsage (fact) {
+    if ((this.referencePaths[fact.fact]) && (this.referencePaths[fact.fact].length === 1)) {
+      const node = jsonc.findNodeAtLocation(this.tree, this.identifierPaths[fact.fact])
+      const beginPosition = node.offset
+      const endPosition = node.offset + node.length
+      const path = jsonc.getNodePath(node)
+      return [{
+        code: 'LR0004',
+        message: 'Unused fact: ' + fact.fact,
```

Comment:
This kind of duplicates the information in the 'source' attribute. In the end in vscode this would show as something like: "Unused fact [unusedfact] ([unusedfact])", so I would remove the fact.fact in the message, and just have 'Unused fact'